src/routes/scheduleContractsDreSync.js

```diff
@@ -0,0 +1,86 @@
+const { config } = require('../config');
+const { LoggerFactory } = require('warp-contracts');
+const { Queue } = require('bullmq');
+const { getAllContractsIds } = require('../db/nodeDb');
+
+const logger = LoggerFactory.INST.create('scheduleContractsDreSync');
+const isTestInstance = config.env === 'test';
+
+module.exports = async (ctx) => {
+  if (!config.foreignDres || Object.keys(config.foreignDres).length < 1) {
+    logger.info('Cannot sync with remote dres, config missing');
+    ctx.body = 'Foreign DREs sync functionality is disabled';
+    ctx.status = 404;
+    return;
+  }
+
+  const foreignDre = ctx.query.dre;
+  if (!foreignDre || !config.foreignDres[foreignDre]) {
+    logger.info('Cannot sync with remote dre, invalid name');
+    ctx.body = `${foreignDre} not recognized, maybe try: /sync-contracts?dre=dreu&size=2`;
+    ctx.status = 400;
+    return;
+  }
+
+  const { nodeDb } = ctx;
+  const size = ctx.query.size;
+
+  try {
+    const difference = await Promise.all([fetchRemoteContracts(foreignDre), getAllContractsIds(nodeDb)]).then(
+      ([foreign, local]) => foreign.ids.filter((x) => !local.ids.includes(x))
+    );
+
+    const registerQueue = new Queue('register', {
```

Comment:
This will reuse the `register` queue. 
At least that is what I hope for...
"When you instance a Queue, BullMQ will just upsert a small "meta-key", so if the queue existed before it will just pick it up and you can continue adding jobs to it."
https://docs.bullmq.io/guide/queues

The queue data is back in the status endpoint
```json
"workersConfig": {
"register": 1,
"update": 1,
"signature": 2,
"jobIdRefreshSeconds": 5,
"maxFailures": 3,
"maxStateSizeB": 1073741824
},
"queues_totals": {
"register": {
"active": 0,
"waiting": 0
}
},
"queues_details": {
"register": {
"active": [],
"waiting": []
}
}
```